src/iosMain/kotlin/org/cru/godtools/tool/xml/IosXmlPullParser.kt

```diff
@@ -27,11 +38,20 @@ class IosXmlPullParser(parser: NSXMLParser) : SaxXmlPullParser() {
         override fun parserDidEndDocument(parser: NSXMLParser) = enqueueEndDocument()
 
         private fun Map<Any?, *>.convert() =
-            map { QName(local = it.key?.toString().orEmpty()) to it.value?.toString().orEmpty() }.toMap()
+            map { it.key?.toString().orEmpty().attrToQName() to it.value?.toString().orEmpty() }.toMap()
+
+        private fun String.attrToQName(): QName = when {
+            !contains(':') -> QName(local = this)
+            else -> {
+                val (ns, local) = split(':', limit = 2)
+                QName(namespaces[ns]!!.last(), local)
+            }
+        }
```

Comment:
IOS doesn't parse attribute namespaces for us :(

so we had to implement our own namespace parsing for attributes